lambda/matching.ml

```diff
@@ -1717,10 +1717,12 @@ let make_constr_matching p def ctx = function
           match cstr.cstr_tag with
           | Cstr_constant _
           | Cstr_block _ ->
-              make_field_args p.pat_loc Alias arg 0 (cstr.cstr_arity - 1) argl
+              make_field_args (of_raw_location p.pat_loc)
+                Alias arg 0 (cstr.cstr_arity - 1) argl
```

Comment:
> Wiring the scopes parameter though the pattern matching code seems particularly easy since it won't change at all.

The amount of work involved here has nothing to do with how much it changes!